acceptance-tests/build.sbt

```diff
@@ -21,7 +21,7 @@ name := "renku-acceptance-tests"
 version := "0.1.0"
 scalaVersion := "2.13.2"
 
-parallelExecution in Test := true
```

Comment:
I can't remember what you said in your presentation. What was the change that required us to run this in sequence?

Reply:
I forgot to mention that but in the end, I had to leave it set to `false` as either selenium or chromium or the whole set-up was failing intermittently when run in parallel.